packages/kbn-config-schema/src/types/type.ts

```diff
@@ -33,6 +33,10 @@ export interface TypeMeta {
    * Whether this field is deprecated.
    */
   deprecated?: boolean;
+  /**
+   * Release version or date that this route will be removed
```

Comment:
What do you think about adding an example to this doc comment?

```suggestion
   * Release version or date that this route will be removed
   * @example 9.0.0
```